api/envoy/extensions/rbac/custom_library_config/v3/custom_library.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package envoy.extensions.rbac.custom_library_config.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.rbac.custom_library_config.v3";
+option java_outer_classname = "CustomLibraryProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: RBAC CEL Custom Library]
```

Comment:
This comes up in other comments, but being clear on the distinction between the extension type and the example instance we are providing to show the use of this extension type will help reduce ambiguity. Right now, both the extension type and the extension instance have the same name.